client/src/assets/scss/style.scss

```diff
@@ -109,6 +109,23 @@ input[type="checkbox"] {
   }
 }
 
+.add-test-fields-card {
+  margin-top: 20px;
+  margin-bottom: 100px;

```

Comment:
This correct? feel a bit weird

Reply:
check the sc, top is to match figma a litttle and bottom is so the drop down expands more